packages/vite/src/node/plugins/css.ts

```diff
@@ -245,6 +246,10 @@ function encodePublicUrlsInCSS(config: ResolvedConfig) {
   return config.command === 'build'
 }
 
+function getLineCount(str: string) {
+  return str.split(splitRE).length - 1
```

Comment:
```suggestion
  return str.match(splitRE).length
```
I think this is slightly better as it won't allocate long strings.

Reply:
Well, I think my previous attempt was incorrect after all. The version now is accurately calculating the lines back.